src/shared/utils.ts

```diff
@@ -443,32 +443,34 @@ export function getMomentLanguage(): string {
 }
 
 export function setTheme(theme: string, forceReload: boolean = false) {
-  if (isBrowser() && (theme !== 'browser' || forceReload)) {
-    // This is only run on a force reload
-    if (theme == 'browser') {
-      theme = 'darkly';
-    }
+  if (!isBrowser()) {
+    return;
+  }
+  if (theme === 'browser' && !forceReload) {
```

Comment:
You can combine these.

Reply:
I think it reads easier to have them separate. The first one says "This code is not run in SSR", while the second deals with the logic of default themes and force reloading

Reply:
I'm good with that.